app/services/foreman_ansible/variables_importer.rb

```diff
@@ -96,5 +99,25 @@ def local_variables
     def remote_variables
       proxy_api.all_variables
     end
+
+    def infer_key_type(value)
+      {
+        TrueClass => 'boolean',
+        FalseClass => 'boolean',
+        Integer => 'integer',
+        Float => 'real',
+        Array => 'array',
+        Hash => 'hash'
+      }.reduce('string') { |memo, (key, type)| value.is_a?(key) && memo == 'string' ? type : memo }
```

Comment:
I'm probably missing something, but can't we add String => 'string' to the hash and then just do `hash[value.class] || 'string'`?

Reply:
```ruby
13.class #=> Fixnum
```

I had that previously, rubocop was complaining I should be using Integer instead of Fixnum. Not sure which one is better, let me know your preference.

Reply:
as discussed, this would be the pattern
```
{
'Fixnum' => 'integer',
'Integer' => 'integer'
}
h[value.class.to_s] || 'string'
```

Reply:
I made updates as suggested.